contracts/adapters/dmm/DmmTokenAdapter.sol

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2020 Zerion Inc. <https://zerion.io>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import { ERC20 } from "../../ERC20.sol";
+import { TokenMetadata, Component } from "../../Structs.sol";
+import { TokenAdapter } from "../TokenAdapter.sol";
+
+/**
+ * @dev IDmmToken contract interface.
+ * Only the functions required for DmmTokenAdapter contract are added.
+ * The IDmmToken contract is available here
+ * github.com/defi-money-market-ecosystem/protocol/blob/master/contracts/interfaces/IDmmController.sol.
```

Comment:
this link was meant to be the link to the contract itself (DmmController.sol), not the interface
but you can leave it as is

Reply:
Gotcha. I figured the interface was better because there is documentation in there.

Reply:
Oh, I didn't notice that only interfaces are documented.
So, you can leave links to interfaces)